package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "uncss",
-  "version": "0.16.0",
+  "version": "0.16.1",
```

Comment:
I thought I pushed this? Do you need to rebase?

Reply:
It's already done when merging.